app/controllers/bus_stops_controller.rb

```diff
@@ -0,0 +1,47 @@
+class BusStopsController < ApplicationController
+  def index
+    @bus_route
```

Comment:
needed?

Reply:
No, just throwing things out there for react. Will delete.